apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss

```diff
@@ -580,7 +580,13 @@ span.icon-export-icon {
                justify-content: flex-start !important;
            }
         }
-
+        .invoice-modal-content{
+            left: 60px;
+            width: 100%;
+        }
+        .invoice-modal-content:after{
```

Comment:
Consider using nesting and &:after in .invoice-modal

Reply:
Done